test/polytopes.jl

```diff
@@ -186,9 +186,9 @@
 
     # test accessor methods
     poly = PolyArea(P2[(1,2),(2,3),(1,2)])
-    @test vertices(poly) == CircularVector(P2[(1,2),(2,3)])
+    @test vertices(poly) == Meshes.CircularVector(P2[(1,2),(2,3)])
```

Comment:
You can also add `CircularArrays` to the test/Project.toml and `using CicurlarArrays.jl` in runtests.jl